doc/sphinx/examples_cpp/basics/basics.cpp

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2017 Sebastian Weber, Henri Menke. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <boost/filesystem.hpp>
+#include <stdexcept>
+#include <iostream>
+
+#include "libpairinteraction/dtypes.h"
```

Comment:
We need to rename that header.  This looks weird.  We could adopt something like Eigen, e.g.
```cpp
#include <libpairinteraction/Types>
#include <libpairinteraction/State>
#include <libpairinteraction/System>
```
What do you think?

Reply:
Good idea, I have changed the headers.